### ISSUE TITLE
REGISTEDATE invalid output

### ISSUE THREAD

**Issue by iBreakEverything** [chr233/ASFEnhance#284]
Bug: **REGISTEDATE** invalid output for accounts that don't have the 1 year badge on steam (https://steamcommunity.com/my/badges/1). This will display the info from some other badge
```
>> Enter command: REGISTEDATE ASF
<> Executing...
<bot> 25 of 28 tasks completed. Complete 2 more Steam Community tasks to earn the Level 3 badge.
```

Consider using the dedicated API for it https://developer.valvesoftware.com/wiki/Steam_Web_API#GetPlayerSummaries_(v0002) and extract the data from `.response.players[0].timecreated`. You will get a UNIX timestamp with the exact date and time.

**Comment by chr233** [comment]
thank you

**Comment by chr233** [comment]
seems that api requires api key, use access_token is not allowed.

**Comment by iBreakEverything** [comment]
right, api key feature was removed https://github.com/JustArchiNET/ArchiSteamFarm/issues/3084
guess a message "Your register date is less than 1 year" will do

**Comment by chr233** [comment]
> right, api key feature was removed https://github.com/JustArchiNET/ArchiSteamFarm/issues/3084
> guess a message "Your register date is less than 1 year" will do

asfenhance support to specify an api key in ASF.json, i will add this feature
